R/run_prep_calculation_loans.R

```diff
@@ -1,13 +1,20 @@
 #' Run stress testing data preparation for loans
 #'
+#'
 #' @inheritParams run_stress_test
+#' @param data_prep_output_path Path where results are written.
```

Comment:
I think the argument name is too long. You rarely see this. I think the should be evocative enough that it jogs your memory, but the details should be not in the name but in the helpfile. While I get there is some extra clarity in a long name, the code overall becomes less clear when they force you or the user to wrap a line in an odd place -- before that line becomes coherent in itself.


```r
# Clearer: The entire idea fits in a single line
# For details see `?function_name()` -- as usual
function_name(argument1 = "blah1", argument2 = "blah2")

# Less clear
here_is_a_long_function_name(
   with_this_long_argument1= "blah1", 
   and_this_other_argument2 = "blah2"
)
```

This gets much worse when you consider more than on function in a nested call or a pipe.